server/controllers/userController.js

```diff
@@ -30,6 +50,54 @@ const getAllUserInfo = async (req, res) => {
   }
 };
 
+const getEmail = async (req, res) => {
```

Comment:
Return message is great, but I think it'd be better if it was a string like `"success"` or `"failure"` to indicate that the email was found or not